gerrit/changes/change.py

```diff
@@ -0,0 +1,131 @@
+"""
+Change
+======
+
+Manage gerrit changes
+"""
+
+from gerrit.helper import decode_json
+from gerrit.error import UnhandledError
+from gerrit.projects.project import Project
+
+class Change(object):
+    """Manage gerrit changes"""
+
+    def __init__(self, gerrit_con):
+        self._gerrit_con = gerrit_con
+        self._change_id = None
+        self.full_id = None
+        self.project = None
+        self.branch = None
+        self.change_id = None
+        self.subject = None
+        self.status = None
+        self.created = None
+        self.updated = None
+        self.mergable = None
+        self.insertions = None
+        self.deletions = None
+        self.number = None
+        self.owner = None
+
+    def get_change(self, project, branch, change_id):
+        """
+        Get ChangeInfo for a change
+        :returns: Dict of the ChangeInfo for the change
+        :rtype: dict
+        :exception: ValueError, UnhandledError
+        """
+        if isinstance(project, Project):
+            project = project.name
+
+        if project == '':
+            raise KeyError('Project required')
+
+        if branch == '':
+            raise KeyError('Branch required')
+
+        if change_id == '':
+            raise KeyError('Id required')
+
+        # HTTP REST API HEADERS
+        self._change_id = '%s~%s~%s' % (project, branch, change_id)
+        self._gerrit_con = self._gerrit_con
+
+        r_endpoint = "/a/changes/%s/" % self._change_id
+
+        req = self._gerrit_con.call(r_endpoint=r_endpoint)
+
+        status_code = req.status_code
+        result = req.content.decode('utf-8')
+
+        if status_code == 200:
+            change_info = decode_json(result)
+        elif status_code == 404:
+            raise ValueError(result)
+        else:
+            raise UnhandledError(result)
+
+        self.full_id = change_info.get('id')
+        self.project = change_info.get('project')
+        self.branch = change_info.get('branch')
+        self.change_id = change_info.get('change_id')
+        self.subject = change_info.get('subject')
+        self.status = change_info.get('status')
+        self.created = change_info.get('created')
+        self.updated = change_info.get('updated')
+        self.mergable = change_info.get('mergable')
+        self.insertions = change_info.get('insertions')
+        self.deletions = change_info.get('deletions')
+        self.number = change_info.get('number')
+        self.owner = change_info.get('owner')
+
+        return self
+
+    def add_change(self, project, subject, branch, options):
```

Comment:
Should probably be called create_change?
https://gerrit-review.googlesource.com/Documentation/rest-api-changes.html#create-change